internal/build/info_test.go

```diff
@@ -1,9 +1,78 @@
 package build
 
 import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
 	"testing"
+
+	"os/exec"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/mod/modfile"
 )
 
 func TestGetDetails(t *testing.T) {
 	GetInfo()
 }
+
+// This test ensures that the OpenAPI SHA the CLI is being generated from matches
+// the OpenAPI SHA of the Go SDK version used in the CLI. We should always upgrade
+// the Go SDK version before generating the CLI because downstream generated assets
+// like the bundle schema depend on the Go SDK itself.
+func TestConsistentDatabricksSdkVersion(t *testing.T) {
```

Comment:
Please move this to a separate file as it is not related to `info.go`.

Something like `sdk_consistency_test.go` seems like a better fit.